extension/chrome/elements/pgp_block_modules/pgp-block-render-module.ts

```diff
@@ -46,14 +46,17 @@ export class PgpBlockViewRenderModule {
     });
   };
 
-  public renderContent = (htmlContent: string, isErr: boolean) => {
+  public renderContent = (htmlContent: string, isErr: boolean, isCheckSumInvalid = false) => {
     let contentWithLink = linkifyHtml(htmlContent);
 
     // Temporary workaround for an issue where 'cryptup_reply' divs are not being hidden when replying to all
     // messages from the FES. The root cause is that FES currently returns the body of
     // password message replies as 'text/plain', which does not hide the divs as intended.
     // This workaround can be safely removed once the FES is updated to return the body of message replies as 'text/html'.
     // https://github.com/FlowCrypt/flowcrypt-browser/issues/5004
+    if (isCheckSumInvalid) {
+      contentWithLink = `<div class="pgp-invalid-checksum">${Lang.pgpBlock.invalidCheckSum}</div>${contentWithLink}}`;
+    }
```

Comment:
let's move `if (isCheckSumInvalid) {` check above `temporary workaround` comment, as currently it looks like a part of this workaround and should be removed after FES will send `text/html` messages